src/_pytest/fixtures.py

```diff
@@ -964,6 +1011,8 @@ def __call__(self, function):
                 "fixture is being applied more than once to the same function"
             )
 
+        function = wrap_function_to_warning_if_called_directly(function, self)
```

Comment:
(just double checked, good that this is going to `features`):

changing a decorator from returning the original function to returning a wrapper always seems to break things -- I'm hoping nobody was depending on decorator ordering for `@fixture` before :)